src/date.js

```diff
@@ -63,3 +63,15 @@ export function withinDates (min, max, date) {
   const mDate = moment(date)
   return mDate.isBetween(min, max, 'day') || mDate.isSame(min, 'day') || mDate.isSame(max, 'day')
 }
+
+export function setDateToZeroTime(date) {
+  date.setHours(0)
+  date.setMinutes(0)
+  date.setSeconds(0)
+  return date
+}
+
+export function dateInFuture(current, byDays) {
```

Comment:
You can compose this from `shiftDate` function. You can also get rid of this function since it is being used only in the Timetable component.